app/routes/_extras.brand.tsx

```diff
@@ -0,0 +1,345 @@
+          Drive.
+        </a>
+      </p>
+      <LogoBox name="remix-letter" Logo={LogoRemixLetter} />
+      <AssetHeader>Logo Word</AssetHeader>
+      <LogoBox name="remix" Logo={LogoRemix} />
+    </div>
+  );
+}
+
+function AssetHeader({ children }: { children: React.ReactNode }) {
+  return (
+    <h2 className="text-xl font-extrabold dark:text-gray-200 md:text-3xl">
+      {children}
+    </h2>
+  );
+}
+
+interface LogoProps {
+  fg: string;
+  highlight: string;
+  outline?: string;
+}
+
+function LogoBox({
+  name,
+  Logo,
+}: {
+  name: "remix-letter" | "remix";
+  Logo: FC<LogoProps>;
+}) {
+  // Tailwind classnames for the various ways we style the individual
+  // component pieces of each LogoBox variant
+  let variants = {
+    light: {
+      bg: "bg-white",
+      border: "border-gray-50 dark:border-transparent",
+      fg: "fill-gray-900",
+      highlight: "fill-black-brand",
+      outline: "black",
+    },
+    "light-outline": {
+      bg: "bg-white",
+      border: "border-gray-50 dark:border-transparent",
+      fg: "fill-gray-900",
+      highlight: "fill-transparent",
+      outline: "black",
+    },
+    dark: {
+      bg: "bg-gray-900",
+      border: "border-transparent dark:border-gray-800",
+      fg: "fill-gray-900",
+      highlight: "fill-white",
+      outline: "white",
+    },
+    "dark-outline": {
+      bg: "bg-gray-900",
+      border: "border-transparent dark:border-gray-800",
+      fg: "fill-gray-900",
+      highlight: "fill-transparent",
+      outline: "white",
+    },
+    "glowing": {
+      bg: "bg-gray-900",
+      border: "border-gray-50 dark:border-transparent",
+      fg: "fill-gray-900",
+      highlight: "fill-white",
+      outline: "glow",
+    },
+  };
+  return (
+    <div className="grid grid-cols-2 grid-rows-2 gap-4 gap-x-6">
+      {Object.entries(variants).map(
+        ([variant, { bg, border, fg, highlight, outline }]) => (
+          <div className="flex flex-col" key={variant}>
+            <div
+              className={`flex h-40 items-center justify-center rounded-lg border-[3px] p-4 md:h-48 ${bg} ${border}`}
+            >
+              <Logo fg={fg} highlight={highlight} outline={outline} />
+            </div>
+            <div className="mt-1 flex items-end gap-4 text-sm text-gray-800">
+              {["svg", "png"].map((format) => (
+                <a
+                  className="uppercase underline opacity-50 hover:opacity-100"
+                  href={`/_brand/${name}-${variant}.${format}`}
+                  download={true}
+                  key={format}
+                >
+                  {format}
+                </a>
+              ))}
+            </div>
+          </div>
+        ),
+      )}
+    </div>
+  );
+}
+
+function LogoRemixLetter({ fg, highlight, outline }: LogoProps): JSX.Element {
+  if (outline === "glow") {
+    return glowingLogoRemixLetter({ fg, highlight, outline });
```

Comment:
For these little utils, let's go ahead and just change these to components. I believe Remix's diffing handles it a bit better, but even if not it's a bit simpler in my opinion to just think of everything returning JSX as a component

```tsx
<GlowingLogoRemixLetter {...logoProps} />
```